### PR TITLE
feat: display vfolder in Service detail page

### PR DESCRIPTION
**Pull request by yomybaby**
After:
image

This PR doesn't include the open the folder by clicking on the Service detail page.

---

### Summary
🛠️📁🌐

This pull request adds a feature to show and navigate to the model storage folders for each model service in the web UI. It updates the GraphQL schema, the relay configuration, the web component context, and the routing list page to support this feature. It also adds new translation keys and values for the service detail labels in various languages.

> _`relay.config.js`_
> _Custom scalars for GraphQL_
> _Autumn leaves falling_

### Walkthrough
*  Add a customScalars field to the relay config file to handle the UUID scalar type in the GraphQL schema ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-78a5d0314616c9bfa2e49b1fb473aba3a9418108d13ff3085c5785cc3d8355e3R29-R31))
*  Export the VFolder type from VFolderSelect.tsx and add a new component VFolderLazyView.tsx to render a virtual folder name with an optional link to the data page using react-query and suspense ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-37a0a7c2be6ca99e2f4289ab07f658bc6af9db215c5a77d516d26659070a9717R1-R55), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-d191efffefe30deea0936aba97b2645040e05e1276691079ca15866550abbdefL8-R8))
*  Modify the WebComponentContextType interface and the moveTo function in DefaultProviders.tsx to accept an optional params argument for passing additional parameters to the moveTo event handler ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-1ec6f89edfe2d9ae74244de78c2c81173269c809f471dd0195a10bc02420f860L27-R32), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-1ec6f89edfe2d9ae74244de78c2c81173269c809f471dd0195a10bc02420f860L108-R116))
*  Import the VFolderLazyView component, the LoadingOutlined icon, the Spin component, and the Suspense component in RoutingListPage.tsx to show the model storage folder information for each model service in the service detail panel ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8R9), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8R21), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8R34), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8L41-R44))
*  Add the model and model_mount_destination fields to the modelServiceQuery in RoutingListPage.tsx and add a fallback value for the endpointId variable ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8R129-R130), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8L162-R167))
*  Add a new Descriptions.Item component in RoutingListPage.tsx to render the model storage folder name using the VFolderLazyView component inside a Suspense component and modify the label and span props of the existing Descriptions.Item component that renders the image name ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8L334-R346))
*  Replace the endpoint_id field with the token field as the rowKey prop of the Table component in RoutingListPage.tsx to avoid potential errors or duplicates when rendering the table rows ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8L360-R372))
*  Add a null check for the session variable in RoutingListPage.tsx to avoid calling the openSessionErrorModal function with a null or undefined value ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-b95c8773f0011bade5e876018f34889f34d54b495ccdabe90c2d81a7ca5334e8L459-R473))
*  Import the navigate function, the store, and the globalThis in backend-ai-serving-list.ts and add a moveTo event listener to the backend-ai-react-serving-list component to handle the moveTo event from the VFolderLazyView component and navigate to the data page with the selected virtual folder ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-41a34aa9c45062d3addaba2d47b5ed1e2de7237d854b55fe7f4771db12d6b43cL5-R10), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-41a34aa9c45062d3addaba2d47b5ed1e2de7237d854b55fe7f4771db12d6b43cL50-R63))
*  Add new translation keys and values for the ModelStorage and Image labels in en.json, fr.json, id.json, ko.json, mn.json, and ru.json and mark the values as __NOT_TRANSLATED__ for the languages other than English and Korean ([link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-5295c72e8e9f81f41ad24071df9b7dba8bf9672ff8d5c37d587f374166008f44L283-R284), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-5295c72e8e9f81f41ad24071df9b7dba8bf9672ff8d5c37d587f374166008f44L456-R458), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-cb28ed7aa2d69a36b056c5fc48c220ac0634c320118dd903ffa7b508e6656f24L269-R270), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-cb28ed7aa2d69a36b056c5fc48c220ac0634c320118dd903ffa7b508e6656f24L1522-R1524), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-efa18d69e25ee4acbf51802e5b7d487d283299cbcbbc319388b175bcf1a0426bL269-R270), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-efa18d69e25ee4acbf51802e5b7d487d283299cbcbbc319388b175bcf1a0426bL1522-R1524), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-42a74440ee50b3644e615a3a3c71a4a6ae54dff82f52b5eccf3aaf22b4811b1cR260), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-42a74440ee50b3644e615a3a3c71a4a6ae54dff82f52b5eccf3aaf22b4811b1cL444-R446), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-0e2edcc06e27c33466506878abd4fcbad53e695960932b8de70aedcb6f9fe301L269-R270), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-0e2edcc06e27c33466506878abd4fcbad53e695960932b8de70aedcb6f9fe301L1522-R1524), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-00509cee1359d9305e94d87843979c41295c5768c2e4b0b7f07519c341fd7ff7L269-R270), [link](https://github.com/lablup/backend.ai-webui/pull/1924/files?diff=unified&w=0#diff-00509cee1359d9305e94d87843979c41295c5768c2e4b0b7f07519c341fd7ff7L1522-R1524))

### CODE REVIEW

File: react/src/pages/RoutingListPage.tsx [hunk 1]
```diff
@@ -331,7 +335,15 @@ const RoutingListPage: React.FC<RoutingListPageProps> = () => {
               )}
             </Flex>
           </Descriptions.Item>
-          <Descriptions.Item label="Image">
+          <Descriptions.Item label={t('session.launcher.ModelStorage')}>
+            <Suspense fallback={<Spin indicator={<LoadingOutlined spin />} />}>
+              {endpoint?.model && (
+                <VFolderLazyView uuid={endpoint?.model} clickable={false} />
+              )}
+              ㄱ
```
Comment (agatha197) [inline react/src/pages/RoutingListPage.tsx]:
```suggestion
```

Comment (agatha197) [reply react/src/pages/RoutingListPage.tsx]: I found typo errors ;)
image

**Review by agatha197 — CHANGES_REQUESTED** [review]

File: react/src/pages/RoutingListPage.tsx (hunk 1 above)
Comment (yomybaby) [reply react/src/pages/RoutingListPage.tsx]: 😱

File: resources/i18n/en.json [hunk 2]
```diff
@@ -280,7 +280,8 @@
       "PreOpenPortConfigurationDone": "Preopen ports configured successfully.",
       "PreOpenPortPanelTitle": "Preopen ports to set (optional)",
       "PreOpenPortRange": "Preopen ports are only available from 1024 to 65535.",
-      "MinMemory": "The minimum memory capacity for the currently selected runtime environment is {{size}}iB."
+      "MinMemory": "The minimum memory capacity for the currently selected runtime environment is {{size}}iB.",
+      "ModelStorage": "ModelStorage"
```
Comment (agatha197) [inline resources/i18n/en.json]:
```suggestion
      "ModelStorage": "Model Storage"
```

**Review by agatha197 — CHANGES_REQUESTED** [review]
Last one minor issue!

**Review by agatha197 — APPROVED** [review]
LGTM!

**Pull request merged**